src/location/locations.ts

```diff
@@ -271,8 +292,10 @@ function buildSiblingDecorations(item: LocationItem, document: vscode.TextDocume
 }
 
 function hasSameNullSafeResourceUri(f1: FileItem, f2: FileItem) {
-  return f1.resourceUri === null && f2.resourceUri === null
-    || f1.resourceUri !== null && f2.resourceUri !== null && f1.resourceUri.toString() === f2.resourceUri.toString();
+  return (
```

Comment:
These are only formatting changes. They are made automatically whenever I make changes in any file, so sorry I did not separate them into a separate commit.